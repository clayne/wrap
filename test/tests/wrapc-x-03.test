wrapc | /dev/null | -x | wrapc-x-03.c | 0
