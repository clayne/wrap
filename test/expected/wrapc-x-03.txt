/**
 * BLANK LINE BELOW
 *
 * @parblock
 * FOO
 * @endparblock
 *
 * BLANK LINE ABOVE
 * @return Returns 0.
 */
